ambari-server/src/main/java/org/apache/ambari/server/upgrade/UpgradeCatalog270.java

```diff
@@ -854,7 +859,7 @@ protected void addAmbariConfigurationTable() throws SQLException {
     List<DBAccessor.DBColumnInfo> columns = new ArrayList<>();
     columns.add(new DBAccessor.DBColumnInfo(AMBARI_CONFIGURATION_CATEGORY_NAME_COLUMN, String.class, 100, null, false));
     columns.add(new DBAccessor.DBColumnInfo(AMBARI_CONFIGURATION_PROPERTY_NAME_COLUMN, String.class, 100, null, false));
-    columns.add(new DBAccessor.DBColumnInfo(AMBARI_CONFIGURATION_PROPERTY_VALUE_COLUMN, String.class, 255, null, true));
+    columns.add(new DBAccessor.DBColumnInfo(AMBARI_CONFIGURATION_PROPERTY_VALUE_COLUMN, String.class, 2048, null, true));
```

Comment:
This won't actually add the correct types as you have defined them in the SQL files. For example, String.class is bound to a `VARCHAR` for MySQL. However, you have it set to a `MEDUIMTEXT` type down below. On upgrades, the column will be `VARCHAR(2028)` but on new installs it will be a `MEDUIMTEXT` capable of holding 16MB of data.

Reply:
I will change the DDL to be `VARCHAR(2048)` rather than `MEDUIMTEXT` to be consistent.  `VARCHAR(2048)` should be plenty of room for now.